test/runtests.jl

```diff
@@ -1,8 +1,7 @@
-
+include("../src/AutoHashEquals.jl")
```

Comment:
you shouldn't usually do this, it's assumed that the package is on load path somewhere if you're running this file